src/test/python/agents/atba/atba.py

```diff
@@ -50,8 +50,12 @@ def load_config(self, config_header):
     @staticmethod
     def create_agent_configurations():
         config = super(Atba, Atba).create_agent_configurations()
-        config.add_header_name('Bot Parameters').add_value('flip_turning', bool, default=False,
-                                                           description='if true bot will turn opposite way')
+        config.get_header('Bot Parameters').add_value('flip_turning', bool, default=False,
+                                                      description='if true bot will turn opposite way')\
```

Comment:
can save the header and then apply it on that instead of chaining.
I like to avoid the \ in python